github/src/main/java/com/redhat/ipaas/github/extended/FileContent.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2017 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.redhat.ipaas.github.extended;
+
+import java.util.Base64;
+
+/**
+ * Used for JSON serialization of a file content entry
+ */
+class FileContent {
```

Comment:
Would prefer at least fine getters! Assume this is using private fields for json serialisation right now? confused me!